base/rational.jl

```diff
@@ -4,7 +4,27 @@
     Rational{T<:Integer} <: Real
 
 Rational number type, with numerator and denominator of type `T`.
-Rationals are checked for overflow.
+
+The preferred way of creating `Rational` numbers is using the
+`//` operator. `Rational`s are also checked for overflow.
+
+`Rational(x)` where `x` is a subtype of `AbstractFloat` returns
+the exact conversion of `x` to a `Rational` number.
```

Comment:
This should go in a docstring for the `Rational(::AbstractFloat)` method, not for the `Rational` type.

Reply:
OKAY